app/routes/company/components/CompaniesPage.js

```diff
@@ -30,7 +30,12 @@ const CompanyItem = ({ company }: Company) => {
         <div className={styles.companyLogoContainer}>
           <Link to={`/companies/${company.id}`}>
             <div className={styles.companyLogo}>
-              {<Image src={company.logo} />}
+              {
+                <Image
+                  src={company.logo}
+                  placeholder={company.logoPlaceholder}
+                />
+              }
```

Comment:
I realize the curly bois were there already { }, but why?

Reply:
hmmm good question. :shrug: